compiler/src/dotty/tools/dotc/parsing/Parsers.scala

```diff
@@ -3691,6 +3714,14 @@ object Parsers {
         template(emptyConstructor)
       r
 
+    /** with Template, with EOL <indent> interpreted */
+    def withTemplate(constr: DefDef, parents: List[Tree]): Template =
+      if in.token != WITHEOL then accept(WITH)
+      possibleTemplateStart() // consumes a WITHEOL token
```

Comment:
Do we miss `else` here?
```suggestion
      else possibleTemplateStart() // consumes a WITHEOL token
```

Reply:
No. `possibleTemplateStart` expects a `{` or a `WITHEOL`.